<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mosaicrClass</class>
 <widget class="QMainWindow" name="mosaicrClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mosaicr</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QRadioButton" name="dirRadioButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>321</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Input all files from directory that match pattern</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="filesRadioButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>361</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Select specific input files</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dirEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="patternEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="filesEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>421</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="mosaicEdit">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>180</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="downEdit">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>220</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="dirButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="filesButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mosaicButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output low-res mosaic</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Downsample ration (factor of 2)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Directory</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pattern (eg mrg*.tif)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>586</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mosaicr.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>dirButton</sender>
   <signal>clicked()</signal>
   <receiver>mosaicrClass</receiver>
   <slot>doDirBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>517</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>518</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filesButton</sender>
   <signal>clicked()</signal>
   <receiver>mosaicrClass</receiver>
   <slot>doFilesBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>523</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>524</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mosaicButton</sender>
   <signal>clicked()</signal>
   <receiver>mosaicrClass</receiver>
   <slot>doMosacBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>512</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>mosaicrClass</receiver>
   <slot>doApply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>mosaicrClass</receiver>
   <slot>doCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>524</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>460</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>doDirBrowse()</slot>
  <slot>doFilesBrowse()</slot>
  <slot>doMosacBrowse()</slot>
  <slot>doApply()</slot>
  <slot>doCancel()</slot>
 </slots>
</ui>
